### ISSUE TITLE
Team Fortress 2: Scorch Shot Afterburn Damage

### ISSUE THREAD

**Issue by mrw-b** [ValveSoftware/Source-1-Games#6128]
Hello,

I believe the Follow-Up flare from the Scorch Shot applies an incorrect duration of Afterburn. The first flare applies the usual 7.5 seconds of Afterburn (total 60 damage), but  the Follow-Up flare will apply Afterburn which lasts for 10 seconds (total 80 damage). I think this is an oversight from Jungle Inferno, when flare gun Afterburn was reduced from 10 seconds to 7.5 seconds, and only the original flare was updated while the Follow-Up flare was left at 10 seconds. There are YouTube links at the end.

This allows Pyro to one-shot the 125hp classes as the pyro will deal between 126-134 damage in total if both flares hit and leads to a large amount of frustration for players in the community. I imagine this wasn't the design as it allows pyro to kill players way outside his intended range with ease and very little thought. 

Best regards

Video 1: https://youtu.be/GvL1Ko1rwvc
Video 2: https://youtu.be/x8iGLiMD_eY

**Comment by trySu** [comment]
(random player responding)
In general crit\mini crit damage applies longer afterburn.
Most flare guns deal crit\mini crit damage to burning players.
In your second video, only the engineer got longer afterburn because he was set on fire by the first hit, so the explosion was a mini crit on him (same with the medic in video 1).
The afterburn is applied correctly.
I do understand the frustration though and it might fit under the "feature request" category.
Have a great day.

Edit: my info is false (crits\mini crits don't give extra afterburn time) and from seeing other replies - it's just that the afterburn time of 7.5s is always added to other fire sources, flames at minimum give 3.5s afterburn so the sum always reaches the maximum of 10s (unless you wait for the timer to reach < 2.5s)
In your example the second impact of the scorch shot will always add up with the first impact so from the first 7.5s it adds up with another 7.5s to reach the 10s mark.
Your question would now be whether the afterburn stacking was intentional or not (though it doesn't seem to be a bug)
Or you would feature request a change, so scorch shot specifically won't reach 10s afterburn so easily.

**Comment by mrw-b** [comment]
> (random player responding) In general crit\mini crit damage applies longer afterburn. Most flare guns deal crit\mini crit damage to burning players. In your second video, only the engineer got longer afterburn because he was set on fire by the first hit, so the explosion was a mini crit on him (same with the medic in video 1). The afterburn is applied correctly. I do understand the frustration though and it might fit under the "feature request" category. Have a great day.

Makes sense. Though, I am now wondering whether (mini-)crit damage having an increased duration is instead unintended as there is no mention of this in the patch notes for Jungle Inferno. Either way, thanks for the info :)

**Comment by jh34ghu43gu** [comment]
From the wiki: "While afterburn duration may stack from multiple sources and players"
Since 10 seconds is the maximum, and the bouncing flare has to hit within ~3 seconds of the initial ignition, maximum afterburn will always be applied (5+7.5 > 10).

Here is a video showing that a bouncing flare hitting a player with <2 seconds of afterburn left does not receive the full 10 seconds worth if they are hit by the bouncing flare: https://youtu.be/7wYDM9xaeiQ

**Comment by mrw-b** [comment]
> From the wiki: "While afterburn duration may stack from multiple sources and players" Since 10 seconds is the maximum, and the bouncing flare has to hit within ~3 seconds of the initial ignition, maximum afterburn will always be applied (5+7.5 > 10).
> 
> Here is a video showing that a bouncing flare hitting a player with <2 seconds of afterburn left does not receive the full 10 seconds worth if they are hit by the bouncing flare: https://youtu.be/7wYDM9xaeiQ

Oh cool. Thanks :)

**Comment by Jeffonehundred** [comment]
> Replying to https://github.com/ValveSoftware/Source-1-Games/issues/6128#issue-2393964141

I play pryo and fight him in meny different matchs, I have no problem with the scorch shot what so ever. Please, if someone from valve is listen. Increase the knock-back that it give to other players. Its damage is fine please dont change that.
